protobuf/v1/snapshot.proto

```diff
@@ -88,3 +89,19 @@ message NexusCreateSnapshotReplicaStatus {
     string replica_uuid = 1; // uuid of the replica
     uint32 status_code = 2;  // snapshot operation status (0 means successful completion)
 }
+
+/// Request Parameters for Creating Snapshot Clone.
+message CreateSnapshotCloneRequest {
+    string snapshot_uuid = 1; // uuid of the snapshot
+    string clone_name = 2; // snapshot clone name
+    string clone_uuid = 3; // uuid of the clone to be created from snapshot
+}
+
+/// Request Parameters for List Snapshot Clone.
+message ListSnapshotCloneRequest {
+  string snapshot_uuid = 1;  // uuid of the snapshot
```

Comment:
make it optional so we can list all clones?
Also should we add an optional clone_uuid as well? Or maybe not needed, can simply ListReplicas with the clone_uuid right?